supabase/migrations/20260901153326_observe_app_nav.sql

```
@@ -0,0 +1,3 @@
+-- SPA / Capacitor navigation events for Observe (no Expo Router required).
+-- Plugins report action=app_nav with metadata.route (or path) and optional duration_ms.
```

SQLFluff (4.3.0)

[error] Line is too long (88 > 80).

(LT05)